.github/workflows/e2e.yml

```diff
@@ -29,9 +29,6 @@ jobs:
         with:
           submodules: recursive
 
-      - name: remove submodule locks
```

Comment:
Is checkout with submodules, above, still relevant? Can't recall if we have foundry now as a submodule

Reply:
yes, still needed for foundry

Reply:
you can see it in `.gitmodules`